webpack/components/extensions/HostDetails/Tabs/ModuleStreamsTab/ModuleStreamsTab.js

```diff
@@ -0,0 +1,228 @@
+
+  const emptyContentTitle = __('This host does not have any Module streams.');
+  const emptyContentBody = __('Module streams will appear here when available.');
+  const emptySearchTitle = __('Your search returned no matching Module streams.');
+  const emptySearchBody = __('Try changing your search criteria.');
+  const { searchParam } = useUrlParams();
+  const columnHeaders = [
+    __('Name'),
+    __('State'),
+    __('Stream'),
+    __('Installation status'),
+    __('Installed profile'),
+  ];
+
+  const fetchItems = useCallback(
+    (params) => {
+      if (!hostId) return hostIdNotReady;
+      return getHostModuleStreams(
+        hostId,
+        params,
+      );
+    },
+    [hostId],
+  );
+
+  const response = useSelector(selectModuleStream);
+  const { results, ...metadata } = response;
+  const status = useSelector(state => selectModuleStreamStatus(state));
+  /* eslint-disable no-unused-vars */
+  const {
+    selectOne, isSelected, searchQuery, selectedCount, isSelectable,
+    updateSearchQuery, selectNone, fetchBulkParams, ...selectAll
+  } = useBulkSelect({
+    results,
+    metadata,
+    isSelectable: _result => false,
+    initialSearchQuery: searchParam || '',
+  });
+  /* eslint-enable no-unused-vars */
+
+  if (!hostId) return <Skeleton />;
+
+  const rowActions = [
+    {
+      title: __('Install'), disabled: true,
+    },
+    {
+      title: __('Update'), disabled: true,
+    },
+    {
+      title: __('Enable'), disabled: true,
+    },
+    {
+      title: __('Disable'), disabled: true,
+    },
+    {
+      title: __('Reset'), disabled: true,
+    },
+    {
+      title: __('Remove'), disabled: true,
+    },
+  ];
+
+  const EnabledIcon = ({ streamText, streamInstallStatus }) => {
+    switch (true) {
+      case (streamText === 'disabled' && streamInstallStatus === 'Not installed'):
+        return <InactiveText text={streamInstallStatus} />;
+      case (streamText === 'unknown' && streamInstallStatus === 'Not installed'):
+        return <InactiveText text={streamInstallStatus} />;
+      case (streamText === 'enabled' && streamInstallStatus === 'Not installed'):
+        return <TableText wrapModifier="nowrap">{streamInstallStatus}</TableText>;
+      case (streamText === 'enabled' && streamInstallStatus === 'Up-to-date'):
+        return <><CheckIcon color="green" /> {streamInstallStatus}</>;
+      case (streamText === 'enabled' && streamInstallStatus === 'Upgradable'):
+        return <LongArrowAltUpIcon color="blue"> {streamInstallStatus}</LongArrowAltUpIcon>;
+      default:
+        return null;
+    }
+  };
+
+  EnabledIcon.propTypes = {
+    streamText: PropTypes.string.isRequired,
+    streamInstallStatus: PropTypes.string.isRequired,
+  };
+
+  const StreamState = ({ moduleStreamStatus }) => {
+    let color;
+    let labelVariant;
+    let streamText = moduleStreamStatus?.charAt(0)?.toUpperCase() + moduleStreamStatus?.slice(1);
```

Comment:
?.charAt(0)?.toUpperCase() < we can probably use the "upperfirst" method to clarify/simplify this.

Reply:
thus can likely be changed to: 
 upperFirst(moduleStreamStatus)